src/demetriek/device.py

```diff
@@ -80,32 +81,35 @@ async def _request(
                     auth=BasicAuth("dev", self.api_key),
                     headers={"Accept": "application/json"},
                     json=data,
-                    raise_for_status=True,
                     ssl=False,
                 )
 
             content_type = response.headers.get("Content-Type", "")
             if "application/json" not in content_type:
                 raise LaMetricError(response.status, {"message": await response.text()})
-            return await response.json()
+
+            response_data: dict[str, Any] = await response.json()
+
+            try:
+                response.raise_for_status()
+            except aiohttp.ClientResponseError as exception:
+                if exception.status in (401, 403):
```

Comment:
If we get here, we already did an `response.json()` call above. Do  401/403 responses contain JSON payloads?

Reply:
Yes—in the case mentioned in the PR description, the `Only notifications with priority 'critical' are allowed in current mode` message is returned as part of the response payload along with a `HTTP 401`.

Reply:
OK 👍